setup.py

```diff
@@ -16,7 +16,7 @@
     long_description_content_type="text/markdown",
     install_requires=[
         "Flask>=1.0",
-        "mistune>=0.8.4",
+        "mistune==0.8.4",
```

Comment:
Is this change expected? It doesn't seem related to URL change.

Reply:
yeah this is the last version before their v2 which has big big breaking changes. So I pinned to the latest working version (which is the one that we currently use)